aiohue/v2/__init__.py

```diff
@@ -246,3 +247,32 @@ async def fetch_full_state(self) -> None:
             self._sensors.initialize(full_state),
             self._groups.initialize(full_state),
         )
+
+    async def get_diagnostics(self) -> Dict[str, Any]:
+        """Return a dict with diagnostic information for debugging and support purposes."""
+        result = {}
+        subst_ids = {}
+
+        def _replace_id(item: dict):
+            """Replace ID with a random one for privacy reasons."""
+            id_key = "id" if "id" in item else "rid"
+            org_item_id = item[id_key]
+            new_id = subst_ids.get(org_item_id, f"redacted-{uuid4()}")
+            subst_ids[org_item_id] = new_id
```

Comment:
uuid4 can be a bit expensive.

```suggestion
						if org_item_id not in subst_ids:
								subst_ids[org_item_id] = f"redacted-{uuid4()}"
            new_id = subst_ids[org_item_id]
```